xml/signals/SPTCO-1930/appearance-DW-2-head.xml

```diff
@@ -114,22 +114,27 @@
     <aspectMapping>
       <advancedAspect>Diverging Clear</advancedAspect>
       <ourAspect>Clear</ourAspect>
+      <ourAspect>Diverging Clear</ourAspect>
```

Comment:
Ah, I just now see why my first comment was confusing.  That last sentence was meant to say that my understanding of how the _changed_ _file_ would work is that it's pointing to the next signal.  Sorry for the poor phrasing. 

I agree with you that "Diverging Clear" is about the track immediately ahead.  But it seems that the quoted bit of logic says "if the next signal (signal in approach) shows Diverging Clear, this signal can show Diverging Clear".  Am I reading that wrong?

(In any case, this change has now passed tests and can be merged; these comments are more about my understanding than a request not to merge)